src/main/java/com/github/webicitybrowser/spec/fetch/TaskDestination/GlobalObject.java

```diff
@@ -0,0 +1,4 @@
+package com.github.webicitybrowser.spec.fetch.TaskDestination;
+
+public class GlobalObject implements TaskDestination{
```

Comment:
I believe it is the GlobalObject's event loop that will implement TaskDestination, not the GlobalObject itself
For modularity reasons, it might be better to have an adapter as opposed to directly implementing, anyways

Reply:
I definitely have to dig further about that in specs